samples/typescript_nodejs/58.teams-start-new-thread-in-channel/package.json

```diff
@@ -18,8 +18,8 @@
         "url": "https://github.com"
     },
     "dependencies": {
-        "botbuilder": "~4.10.1",
-        "botframework-schema": "^4.10.1",
+        "botbuilder": "~4.10.3",
+        "botframework-schema": "^4.10.3",
```

Comment:
Unnecessary dependency as `botbuilder` exposes `botbuilder-core` which exposes `botframework-schema`

Reply:
Created linked issue to track addressing this. https://github.com/microsoft/BotBuilder-Samples/issues/2741